frontend/src/components/tasks/TaskDetailsPanel.tsx

```diff
@@ -117,87 +118,137 @@ export function TaskDetailsPanel({
         <TabNavContext.Provider value={{ activeTab, setActiveTab }}>
           <ProcessSelectionProvider>
             <ReviewProvider>
-              {/* Backdrop - only on smaller screens (overlay mode) */}
-              {!hideBackdrop && (
+              <EntriesProvider>
```

Comment:
Consider wrapping only the `TodoPanel` and `LogsTab` rather than the entire panel to reduce unrelated re-renders on log updates.